system-test/test.sh

```diff
@@ -29,7 +33,12 @@ cp -r "$PWD/system-test/busybench" "$TESTDIR"
 cd "$TESTDIR/busybench"
 
 retry npm install pify @types/pify typescript gts @types/node >/dev/null
+if [[ -z "$BINARY_HOST" ]]; then
```

Comment:
Instead of the if condition you could do
```
${BINARY_HOST:+--pprof_binary_host_mirror=$BINARY_HOST}
```

Example:
```
$ export BINARY_HOST=baz && bash -c "echo --foo ${BINARY_HOST:+--pprof_binary_host_mirror=$BINARY_HOST} --bar"
--foo --pprof_binary_host_mirror=baz --bar

$ unset BINARY_HOST && bash -c "echo --foo ${BINARY_HOST:+--pprof_binary_host_mirror=$BINARY_HOST} --bar"
--foo --bar
```

Reply:
Works with empty values too:
```
$ export BINARY_HOST= && bash -c "echo --foo ${BINARY_HOST:+--pprof_binary_host_mirror=$BINARY_HOST} --bar"
--foo --bar
```

Reply:
Done.